core/modules/widgets/edit-text.js

```diff
@@ -212,17 +212,30 @@ EditTextWidget.prototype.updateEditorDomNode = function(text) {
 };
 
 /*
+Get the first parent element that has scrollbars or use the body as fallback.
+*/
+EditTextWidget.prototype.getScrollContainer = function(el) {
+	while(el.parentNode) {	
+		el = el.parentNode;
+		if(el.scrollTop) {
+			return el;
+		}
+	}
+	return document.body;
+};
+
+/*
 Fix the height of textareas to fit their content
 */
 EditTextWidget.prototype.fixHeight = function() {
-	var self = this,
-		domNode = this.domNodes[0];
+	var domNode = this.domNodes[0];
 	if(this.editAutoHeight && domNode && !domNode.isTiddlyWikiFakeDom && this.editTag === "textarea") {
 		// Resize the textarea to fit its content, preserving scroll position
-		var scrollPosition = $tw.utils.getScrollPosition(),
-			scrollTop = scrollPosition.y;
-		// Measure the specified minimum height
-		domNode.style.height = self.editMinHeight;
+		// Get the scroll container and register the current scroll position
+		var container = this.getScrollContainer(domNode),
+			scrollTop = container.scrollTop;
+                // Measure the specified minimum height
+		domNode.style.height = this.editMinHeight;
```

Comment:
The purpose of the manoeuvre is actually to measure the 'natural' size of the content of the textarea so that we can then explicitly set the textarea itself to that size. If the user is manually sizing the textarea then none of this is needed.